api/src/pcapi/routes/backoffice/finance/forms.py

```diff
@@ -37,15 +37,20 @@ class Meta:
 
     origin = fields.PCStringField("Origine de la demande")
 
+
+class CollectiveIncidentCreationform(BaseIncidentCreationForm):
```

Comment:
Une p'tite majuscule :
```suggestion
class CollectiveIncidentCreationForm(BaseIncidentCreationForm):
```